fews_3di/simulation.py

```diff
@@ -420,6 +434,19 @@ def _download_results(self):
                 f.write(response.content)
             logger.info("Downloaded %s", target)
 
+    def _process_basic_lizard_results(self):
+        self.simulations_api.simulations_results_post_processing_lizard_basic_create(
+            simulation_pk=self.simulation_id,
+            data={
+                "scenario_name": self.settings.results_scenario_name,
+                "process_basic_results": True,
+            },
+        )
+        logger.info(
+            "Basic lizard results will be processed as %s",
+            self.settings.results_scenario_name,
+        )
```

Comment:
Volgens mij wordt er elders gekeken naar het resultaat van zo'n API call. Vaak zit daar ook een URL bij, die is beter om als logger.info() terug te geven, lijkt me.